tests/storage/memorydump.go

```diff
@@ -97,21 +97,10 @@ var _ = SIGDescribe("Memory dump", func() {
 
 	createAndStartVM := func() *v1.VirtualMachine {
 		By("Creating VirtualMachine")
-		vm := libvmi.NewVirtualMachine(libvmifact.NewCirros(), libvmi.WithRunStrategy(v1.RunStrategyAlways))
```

Comment:
Good point. So my idea was to standardize this and use the same starting method as much as possible. I also believe this improves readability, but is probably less efficient.

Reply:
I think it is discouraged to create a vm and then start it if it can be done in one call.
With that said, I get how hard it is to merged this kind of big PRs, so I dont mind to let it slide and handled in a followup PR